cg/cli/set.py

```diff
@@ -145,10 +149,23 @@ def sample(context, sex, customer, comment, downsampled_to, apptag, capture_kit,
 
     if capture_kit:
         sample_obj.capture_kit = capture_kit
-        click.echo(click.style(f"Capture kit {capture_kit} added to sample "
+        click.echo(click.style(f"Capture kit {capture_kit} set on sample "
                                f"{sample_obj.internal_id}", fg='green'))
         context.obj['status'].commit()
 
+    if data_analysis:
+        sample_obj.comment = sample_obj.comment + f"Data-analysis changed from" \
```

Comment:
This will concat the comments without a separator. Can we add an '\r' or a space?

Reply:
I'd like to use join here but  maybe later

Reply:
I think my answer was a besides the point, I'll use a separator :)